core/src/main/resources/struts-default.xml

```diff
@@ -388,6 +390,15 @@
                 <interceptor-ref name="actionMappingParams"/>
                 <interceptor-ref name="params"/>
                 <interceptor-ref name="conversionError"/>
+                <interceptor-ref name="coepInterceptor">
+                    <param name="enforcingMode">true</param>
```

Comment:
This interceptor should *not* be enforcing by default. Existing applications that have cross-origin endpoints will break when they upgrade to the next version if we do so. This value should be either be false here or this interceptor should be removed from the default stack.